testing/tests/loop_else.rs

```diff
@@ -0,0 +1,19 @@
+use askama::Template;
+
+#[derive(Template)]
+#[template(
+    source = "{% for v in values %}{{v}}{% else %}empty{% endfor %}",
```

Comment:
Common style in this project is to keep spaces around expression in `{{ v }}`.

Reply:
Done.